### PR TITLE
Address Ben's IESG comments

### PR DESCRIPTION
**Pull request by wkumari**
This addresses Ben's comments at: https://datatracker.ietf.org/doc/conflict-review-google-self-published-geofeeds/ballot/

### CODE REVIEW

File: draft.xml [hunk 1]
```diff
@@ -442,7 +444,8 @@
         notification, consumers SHOULD implement mechanisms to periodically
         refresh local copies of feed data. In the absence of any other refresh
         timing information, it is recommended that consumers SHOULD refresh
-        feeds no less often than weekly.</t>
+        feeds no less often than weekly, and no more often than is likely
```
Comment (ekline) [inline draft.xml:447]: "is likely TO cause issues"

**Review by ekline — COMMENTED** [review]
Otherwise LGTM

File: draft.xml (hunk 1 above)
Comment (wkumari) [reply draft.xml:447]: Thanks. done.

**Review by wkumari — COMMENTED** [review]
Done, and submitting.

**Pull request merged**